cli/hooks.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/python
 from __future__ import print_function
-from cli.utils import Utils
 import os
 import contextlib
 import sys
+from sets import Set
 from datetime import datetime
+from cli.webhook import WebHook
+from cli.utils import Utils
```

Comment:
This is based on this pep-8 standard recommendation. 
https://www.python.org/dev/peps/pep-0008/#imports. However I am adding an extra line space  between two groups.